src/config.js

```diff
@@ -1,8 +1,12 @@
 export default {
-  GOAL_BUFFER_LENGTH: 60,
+  GOAL_BUFFER_LENGTH: 30,
+  MAX_GOAL_BUFFER_LENGTH: 60,
+  GOAL_BUFFER_RATE: 1,
   // A fudge factor to apply to advertised playlist bitrates to account for
   // temporary flucations in client bandwidth
   BANDWIDTH_VARIANCE: 1.2,
   // How much of the buffer must be filled before we consider upswitching
-  BUFFER_LOW_WATER_LINE: 30
+  BUFFER_LOW_WATER_LINE: 0,
+  MAX_BUFFER_LOW_WATER_LINE: 30,
+  BUFFER_LOW_WATER_RATE: 1
```

Comment:
Do we want separate rates for low waterline and goal buffer length? Or a single rate that they both increase at?

Another note, maybe instead of defining a rate so that we scale current time with and add to the min, what about defining a max, and a time at which the max should be reached. and just interpolate. e.g. with wanting to reach max at 30 seconds, `let t = Math.min(currentTime / 30, 1); GBL = (1 - t) * 30 + t * 60;`

Thoughts @gesinger ?